README.md

```diff
@@ -52,6 +52,29 @@ $ dep ensure github.com/pkg/errors@^0.8.0
 
 See the help text for more detailed usage instructions.
 
+## Semantic Versioning
+
+`dep ensure` a uses an external [semver library](https://github.com/Masterminds/semver) to interpret the version constraints you specify in the manifest. The comparison operators are:
+
+* `=`: equal (aliased to no operator)
```

Comment:
`dep` uses different defaults than the library, so "(aliased to the no operator)" is not correct and should be removed.